src/sentry/api/serializers/models/filechange.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry.api.serializers import Serializer, register
+from sentry.models import Commit, CommitFileChange
+from sentry.api.serializers.models.release import get_users_for_commits
+
+
+@register(CommitFileChange)
+class CommitFileChangeSerializer(Serializer):
+    def get_attrs(self, item_list, user):
+        commits = Commit.objects.filter(id__in=[f.commit_id for f in item_list]).select_related('author')
+        author_objs = get_users_for_commits(commits)
+        commits_by_id = {commit.id: commit for commit in commits}
+        result = {}
+        for item in item_list:
+            result[item] = {
+                'user': author_objs.get(commits_by_id[item.commit_id].author_id, {}),
```

Comment:
Last minor thing, and sorry for not mentioning this before:

Let's do:

```python
commit = commits_by_id[item.commit_id]
result[item] = {
  'user': author_objs.get(commit.author_id, {}),
...
```

That way we're only pulling out of the dictionary once instead of twice.